tools/windows-wsl-local-node-config/README.md

```diff
@@ -0,0 +1,100 @@
+Windows WSL Local Node Config Creation Tool
+========================
+
+**Run a single command to create node configuration files to enable running of a local network for ot-node development.**
+
+This tool will generate configuration files for mulitple OriginTrail nodes, allowing you to run a multi node network locally.
+
+**Note:** This tool is intended for creating development nodes only, running on **Windows Subsystem for Linux (WSL).** For MacOS and Linux, see [MacOS & Linux local network setup]([https://docs.origintrail.io/dkg-v6-upcoming-version/setting-up-your-development-environment#running-multiple-nodes-in-local).
+
+## Prerequisites
+
+* A Windows Subsystem for Linux (WSL) environment running Ubuntu 20.04, and configured for OT-Node development.
+  * See document: **How to setup a Windows development environment** (currently in review)
+
+Usage
+=====
+
+## 1. (Optional) Specify an existing Network Private Key 
```

Comment:
I think there was some misunderstanding here from my part. You can remove this step completely and add the private key to the bootstrap file (same as here https://github.com/OriginTrail/ot-node/blob/v6/develop/tools/local-network-setup/.bootstrap_origintrail_noderc).

Reply:
And also the same for dh_origintrail_noderc, for both files it should work if you use same as it is used for macos tool: https://github.com/OriginTrail/ot-node/blob/v6/develop/tools/local-network-setup/.dh_origintrail_noderc

Reply:
It doesn't really matter what node id they have since they are in local network. This just makes set up and running of the scripts simpler.

Reply:
Thanks both, makes sense.